crypto-auditing/src/event_broker/service.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (C) 2022-2023 The crypto-auditing developers.
+
+//! Internal service definition for the event broker service
+
+use crate::types::EventGroup;
+
+#[tarpc::service]
+pub trait Subscriber {
```

Comment:
Is it safe to make tarpc::service-decorated trait a part of the public API? As in, would moving to a different RPC library cause an API change? Is SubscriberClient also public?